default-config/config

```diff
@@ -34,6 +34,16 @@ InfoStoreAdd terminal xterm
 # This is used for "Run Command" and the Meta+Space key-binding
 InfoStoreAdd runcmd "dmenu_run -nb '#2b4e5e'"
 
+# DesktopConfiguration
+#
+# Global: This is the default setting. In Global mode all screens share desks and pages
+#
+# Per-Desktop: This mode every screen have its own deks and pages. An Pager would be required for each screen.
+#
+# Shared: This mode works in the same maner as Xmonad's or herbsfluftwm handle desks. This mode only works with 2+ screens.
```

Comment:
... `This mode works in the same manner that Xmonad or Herbsflutwm handles desks.  This mode only works with >=2 screens attached.`